modules/appeals_api/spec/requests/appeals_api/v2/decision_reviews/notice_of_disagreements/evidence_submissions_spec.rb

```diff
@@ -134,6 +134,14 @@ def stub_upload_location(expected_location = 'http://some.fakesite.com/path/uuid
       expect(submission['attributes']['appealType']).to eq('NoticeOfDisagreement')
     end
 
+    it 'returns finalStatus for an evidence submission' do
+      es = evidence_submissions.sample
+      get "#{path}#{es.guid}"
+      submission = JSON.parse(response.body)['data']
+
+      expect(submission['attributes']['finalStatus']).to be false
+    end
+
```

Comment:
Can you do the same that you did for Supplemental Claims, and add a test for the `create` endpoint as well?